8Knot/cache_manager/cache_facade.py

```diff
@@ -166,9 +172,17 @@ def caching_wrapper(func_name: str, query: str, repolist: list[int], n_repolist_
     Raises:
         Exception: If a step fails, will print exception and re-raise.
 
-    Returns:
+        Returns:
```

Comment:
Should these spaces be here?

Reply:
this whole thing is a comment so its just a matter of coding style choice. it seems like if we want it to align with "raises" it shouldnt be, but thats just my opinion